core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/Parameter.java

```diff
@@ -40,8 +40,8 @@
 	private String required;
 	@JsonProperty
 	private String defaultValue;
-	@JsonProperty
-	private String choice;
+	@JsonProperty("enum")
+	private String[] choice;
```

Comment:
so the name of the value changed from choice to enum? since when? what about backwards compatibility?